api/src/pcapi/connectors/beneficiaries/ubble.py

```diff
@@ -139,5 +146,21 @@ def get_content(identification_id: str) -> ubble_models.UbbleContent:
     response = session.get(
         build_url(f"/identifications/{identification_id}/"),
     )
+    response.raise_for_status()
+    if not response.ok:
```

Comment:
si not response.ok alors on aura levé une exception au-dessus dans raise_for_status, non ?

Reply:
Dans la lib requests :
```
    @property
    def ok(self):
        """Returns True if :attr:`status_code` is less than 400, False if not.

        This attribute checks if the status code of the response is between
        400 and 600 to see if there was a client error or a server error. If
        the status code is between 200 and 400, this will return True. This
        is **not** a check to see if the response code is ``200 OK``.
        """
        try:
            self.raise_for_status()
        except HTTPError:
            return False
        return True
```
donc c'est clair :-)

Reply:
oui je suis allé trop vite, je voulais surtout qu'on me dise si j'ai mis les logs aux bons endroit
ou si c''était trop bas dans le code